aws/ami/files/scylla_install_ami

```diff
@@ -83,34 +84,17 @@ if __name__ == '__main__':
                 f.write(l)
     run('/opt/scylladb/scripts/scylla_setup --ntp-domain amazon --no-coredump-setup --no-sysconfig-setup --no-raid-setup --no-io-setup --no-bootparam-setup --no-ec2-check')
     run('/opt/scylladb/scripts/scylla_sysconfig_setup --ami')
-    run('/opt/scylladb/scripts/scylla_bootparam_setup --ami')
-    os.remove('/home/centos/.ssh/authorized_keys')
+#XXX: temporary commented out until the script fix for Amazon Linux 2
+#    run('/opt/scylladb/scripts/scylla_bootparam_setup --ami')
```

Comment:
why aren't we running `scylla_bootparam_setup`, is there a scylla issue for that ?

Reply:
@fruch There was a bug that causes an error on scylla_bootparam_setup when we run it on Amazon Linux 2, but it already fixed at https://github.com/scylladb/scylla/commit/4c9369e449623f5efd3676c8954179b7c3c952b8#diff-69301dce68daf68094775d1104243d24, so we can un-comment this now. I'll push the fix.

Reply:
Pushed.